client/dashboard/plugins/plugin/components/action-render-modal-wrapper.tsx

```diff
@@ -0,0 +1,33 @@
+import { Modal } from '@wordpress/components';
+import { ComponentProps } from 'react';
+import ActionRenderModal, {
+	ActionRenderModalProps,
+} from '../../manage/components/action-render-modal';
+
+type ActionRenderModalWrapperProps = ActionRenderModalProps &
+	Pick< ComponentProps< typeof Modal >, 'onRequestClose' | 'title' > & { isOpen: boolean };
+
+export const ActionRenderModalWrapper = ( {
+	actionId,
+	closeModal,
+	isOpen = false,
+	items,
+	onActionPerformed,
+	onExecute,
+	onRequestClose,
+	title,
+}: ActionRenderModalWrapperProps ) => {
+	return (
+		isOpen && (
+			<Modal title={ title } onRequestClose={ onRequestClose }>
+				<ActionRenderModal
```

Comment:
I think the naming is a bit confusing. Would it make more sense to call it `PluginActionRenderModalContentInfo`, similar to how we named other action modals like `site-leave-modal`? If so, we could simplify this one to just `PluginActionRenderModal`.

Additionally, we should try to flatten the folder structure as much as possible, like we did with `/sites`. It would be better to place this under `/plugins` directly and name it something like `plugin-action-modal`.